kuiper-game/Services/CaptainLocator.cs

```diff
@@ -0,0 +1,10 @@
+using Kuiper.Domain;
+
+namespace Kuiper.Services
+{
+    public static class CaptainLocator
+    {
+        public static Captain Captain { get; set; }        
```

Comment:
I kept running into situations where i needed to find the root aggregate and bringing it with me through the entities kept resulting in circular refferences or parameters to ignore. 

Super open for other patterns though!